core/shared/src/main/scala/scalaz/zio/ZSchedule.scala

```diff
@@ -71,7 +75,7 @@ trait ZSchedule[-R, -A, +B] extends Serializable { self =>
         case Nil => IO.succeed(acc)
         case a :: as =>
           self.update(a, s).flatMap {
-            case ZSchedule.Decision(cont, delay, s, finish) =>
+            case ZSchedule.Decision(cont, _, delay, s, finish) =>
```

Comment:
Here i want a suggestion: i'm not considering readiness on run. Should I? If yes, what should be the behavior? Discard results when not ready? Sleep until next readiness?